Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -14,6 +14,8 @@
 MATCH_DEFAULT_VALUE = "any"  # Default "match" value to use when using search filters. Can be either "all" or "any".
 REMOVE_RESPONSE_LINKS = True  # Whether to remove `links` keys from responses.
 REPORT_DOWNLOAD_WAIT_TIME = 60  # Time in seconds to wait before downloading a report after starting its generation
+CONNECTION_ERRORS_RETRIES = 5  # num of times to retry in case of connection-errors
```

Comment:
should it be an argument?

Reply:
hmm, don't think so, doesn't see how its beneficial to customers